kumascript/src/api/util.ts

```diff
@@ -316,6 +311,11 @@ export class HTMLTool {
       // We're here because we can't find the sectionID, so instead we're going
       // to find the live-sample iframe by its id (iframeID, NOT sectionID), and
       // then collect the closest blocks of code for the live sample.
+      console.warn(
+        chalk.yellow(
+          `invalid header id in live sample ${sectionID} within ${this.pathDescription}`
```

Comment:
I'm not sure I understand this error message, so rephrasing to validate my understanding:

```suggestion
          `Live sample references heading id ${sectionID}, but this doesn't exist in ${this.pathDescription}`
```